src/controllers/grade.controller.js

```diff
@@ -29,5 +29,23 @@ class GradesCtrl extends BaseCtrl {
     }
     res.status(httpStatusCodes.OK).send(grade)
   }
+  @put('/:id_grade', auth())
+  async updateGrade(req, res) {
+    let { id_grade: id } = req.params
+
+    let { id: classroomId } = req.params
+    let { name, point } = req.body
+    let grade
+    try {
+      grade = await db.Grade.findOne({ where: { id } })
+      grade.name = name
+      grade.point = point
+      grade.classroomId = classroomId
+      await grade.save()
```

Comment:
This will cause error when there is no record. Double check that case and find solution for it

Reply:
I suggest u can use UPDATE queries , see more: https://sequelize.org/master/manual/model-querying-basics.html#simple-update-queries